site/en/docs/extensions/mv3/getstarted/extensions-101/index.md

```diff
@@ -114,10 +122,25 @@ See [Extension quality guidelines][doc-single-purpose] for additional details.
 
 You can set up a developer account with the [Chrome Web Store][chrome-web-store] to host and
 distribute your extension. Bear in mind that extensions must adhere to the [developer program
-policies][doc-cws-policy]. 
+policies][doc-cws-policy].
 
 See [Publish in the Chrome Web Store][doc-cws-publish] to learn how to distribute your extension.
 
+
+
+{% Details %} 
+{% DetailsSummary %}
+
+**💡 What if I only want to distribute the extension within my organization?**
```

Comment:
```suggestion
**What if I only want to distribute the extension within my organization?**
```

Reply:
I'm not crazy about emojis in formal writing. I also suspect that if emojis had been invented when Strunk and White was written, they would have mentioned them under 'omit needless words.'

Reply:
I like it that it draws attention to a question that developers might be curious about. I have used emojis in the getting started guides as a friendly gesture for new developers. I'd like to keep them for now, and we can meet with the team to present the pros and cons of this approach. 

Let's merge this as is for now, I don't consider it a blocker.